xarray/tests/test_cftime_offsets.py

```diff
@@ -66,22 +66,6 @@ def test_cftime_offset_constructor_valid_n(offset, expected_n):
     assert offset.n == expected_n
 
 
-@pytest.mark.parametrize(
```

Comment:
Based on my comment above, I think we should restore this test.

Reply:
agreed